tools/src/main/scala/scala/scalanative/codegen/AbstractCodeGen.scala

```diff
@@ -70,12 +122,19 @@ private[codegen] abstract class AbstractCodeGen(
     }
   }
 
-  private def genDefns(defns: Seq[Defn])(implicit sb: ShowBuilder): Unit = {
+  private def genDefns(
+      defns: Seq[Defn]
+  )(implicit
+      sb: ShowBuilder,
+      dwf: DebugInformationSection.Builder
+  ): Unit = {
     import sb._
     def onDefn(defn: Defn): Unit = {
       val mn = mangled(defn.name)
       if (!generated.contains(mn)) {
         newline()
+        if (defn.pos != Position.NoPosition)
```

Comment:
You can use `defn.pos.isEmpty` or `defn.pos.isDefined` to make it shorter

Reply:
Done in [f55c49b](https://github.com/scala-native/scala-native/pull/2869/commits/f55c49ba668729a24e5c252226f80f880e2f77f4)